contracts/PushComm/PushCommV3.sol

```diff
@@ -551,154 +510,177 @@ contract PushCommV3 is Initializable, PushCommStorageV2, IPushCommV3, PausableUp
     {
         PUSH_NTT = IERC20(_pushNTT);
         NTT_MANAGER = INttManager(_nttManager);
-        TRANSCEIVER = ITransceiver(_transceiver);
         WORMHOLE_TRANSCEIVER = IWormholeTransceiver(_wormholeTransceiver);
         WORMHOLE_RELAYER = IWormholeRelayer(_wormholeRelayerAddress);
         WORMHOLE_RECIPIENT_CHAIN = _recipientChain;
-        ADD_CHANNEL_MIN_FEES = 50 ether;
     }
 
-    // Cross Chain Request: Create Channel
+    /**
+     * @notice Sets the configuration for core fees
+     * @dev Can only be called by the Push Channel Admin
+     * @param _minChannelCreationFee The minimum fee for creating a channel
+     * @param _feeAmount The amount of the fee
+     */
+    function setCoreFeeConfig(uint256 _minChannelCreationFee, uint256 _feeAmount) external onlyPushChannelAdmin {
```

Comment:
`setCoreFeeConfig` function is missing input validation. A simple validation that input can't be less than 50 PUSH for Add Channel and less than 10 for Fee Amount should be fine for now. @0xNilesh